File: SceytChatUiKit/src/main/java/com/sceyt/sceytchatuikit/presentation/customviews/voicerecorder/RecordingListener.java
package com.sceyt.sceytchatuikit.presentation.customviews.voicerecorder;

public interface RecordingListener {

    void onRecordingStarted();

    void onRecordingLocked();

    void onRecordingCompleted(boolean shouldShowPreview);

    void onRecordingCanceled();
}

File: SceytChatUiKit/src/main/java/com/sceyt/sceytchatuikit/presentation/customviews/voicerecorder/RecordingListener.kt
package com.sceyt.sceytchatuikit.presentation.customviews.voicerecorder

interface RecordingListener {
    fun onRecordingStarted() {}
    fun onRecordingLocked() {}
    fun onRecordingCompleted(shouldShowPreview: Boolean) {}
    fun onRecordingCanceled() {}
}